hydroshare/local_settings.cutover

```diff
@@ -197,10 +184,12 @@ RECAPTCHA_SITE_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
 RECAPTCHA_SECRET_KEY = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
 RECAPTCHA_VERIFY_URL = 'https://www.google.com/recaptcha/api/siteverify'
 
+# insert a google maps key here when in production
+MAPS_KEY = ''
```

Comment:
Should this be in settings.py instead?  A developer is probably never going to add a maps key.  I'm really not sure where the best place to put an empty key is.

Reply:
This was pre-existing and overall, settings.py needs an overhaul (see settings.desired). Can we punt this to a larger cleanup effort.

Reply:
definitely